config/main.py

```diff
@@ -1160,7 +1160,7 @@ def validate_gre_type(ctx, _, value):
         raise click.UsageError("{} is not a valid GRE type".format(value))
 
 
-def multi_asic_save_config(db, filename):
+def multiasic_save_to_singlefile(db, filename):
```

Comment:
>multiasic_save_to_singlefile [](http://example.com/codeflow?start=4&length=28)

what is difference with "show runningconfiguration all"?

Reply:
The difference is that `show run all` will have `bgpraw` data that is not part of ConfigDB